src/screens/add-education/common.ts

```diff
@@ -1,8 +1,8 @@
 const defaultAddEducationPayload = {
-  universityName: '',
-  courseName: '',
+  specialization: '',
+  university: '',
   degree: '',
-  startDate: undefined,
+  startDate: new Date().toISOString(),
```

Comment:
I don't think we need `new Date` here. It's strange to add a course that starts today

Reply:
Undefined does the same in this date input (it will be displayed as current date) @yevhenprybytok told that all dates must be in ISO format